packages/di/README.md

```diff
@@ -114,7 +114,7 @@ test('should return json', async () => {
 
 ### Service level providers
 
-Under the hood, each service decorated with `@injectable` creates its own injector. This means that it is possible to defined providers from that level down.
+Under the hood, each service decorated with `@injectable()` creates its own injector. This means that it is possible to defined providers from that level down.
```

Comment:
typo: `define` -d

nit: I think this is not enough info about provider levels/hierarchies :P